sixtyfps_compiler/tests/syntax/analysis/binding_loop_layout2.60

```diff
@@ -17,4 +41,9 @@ Test := Rectangle {
 //                ^error{The binding for the property 'width' is part of a binding loop}
         }
     }
+
+    w := Wrap { }
+    callback set_wrap();
+    set_wrap => { w.woo = TextWrap.word-wrap; }
```

Comment:
The `Wrap` component here is produce errors because of the `width: height;` in the `Rectangle`, right?

Do I understand correctly that if the callback were invoked, there'd be a binding loop at run-time?

Reply:
correct.  if the text start word-wrapping, the height of the text depends on the width, and so there will be a loop. so that's why we must diagnose the loop at compile time